templates/index.html

```diff
@@ -342,7 +342,8 @@
                      <label><input type="radio" id='express' name="optradio"></label>
                 </div>
              </td>
-         </tr>
+         </tr></form>
+         <form>
```

Comment:
I think, one can remove the "form" But I do not know.

Reply:
I used `form` to get multiple inputs